lib/solaar/ui/config_panel.py

```diff
@@ -76,15 +76,37 @@ def _combo_notify(cbbox, s):
 	c.connect('changed', _combo_notify, setting)
 	return c
 
-# def _create_slider_control(setting):
-# 	def _slider_notify(slider, s):
-# 		if slider.get_sensitive():
-# 			_apply_queue.put(('write', s, slider.get_value(), slider.get_parent()))
-#
-# 	c = Gtk.Scale(setting.choices)
-# 	c.connect('value-changed', _slider_notify, setting)
-#
-# 	return c
+def _create_slider_control(setting):
+	class SliderControl:
+		__slots__ = ('gtk_range', 'timer', 'setting')
+		def __init__(self, setting):
+			self.setting = setting
+			self.timer = None
+
+			self.gtk_range = Gtk.Scale()
+			self.gtk_range.set_range(*self.setting.range)
```

Comment:
Because there is a `@property` `def range()` which returns a tuple, and [`set_range`][1] accepts a `min` and `max` parameter, this works. OK.

 [1]: https://lazka.github.io/pgi-docs/Gtk-3.0/classes/Range.html#Gtk.Range.set_range